fix serveral issues

hugozhu opened the pull request:
issue 1:
when apisix-seed restart empty hosts might be returned, apisix may return 5xx during that peroid.

issue 2:
use 'labels' to tag route with nacos service discovery. 
_discovery_type and _service_name is not compatible in 'upstream'

Commit:
patch etcd value

hugozhu committed:
fixed labels

hugozhu committed:
fixed labels

hugozhu committed:
testing

hugozhu committed:
added discovery_args support

hugozhu committed:
fixed issue when nacos restart, empty hosts

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/api7/apisix-seed?pullRequest=87) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/api7/apisix-seed?pullRequest=87) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: hugozhu<br/>:x: 一粟<br/><hr/>**一粟** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/api7/apisix-seed?pullRequest=87) it.</sub>

hugozhu committed:
added Dockerfile

Commit:
feat: support service use grpc port

Commit:
feat: support service use grpc port v2

Commit:
feat: enhance gRPC port handling in route configuration

- Removed deprecated discovery attributes from test cases.
- Added a new test for marshaling routes with gRPC port support.
- Updated the Marshal function to correctly set the gRPC port for nodes in the route configuration.

hugozhu committed:
Merge pull request #1 from shaoxi/main

feat: support service use grpc port

hugozhu committed:
bump version to 1.1
